config/sources.yml

```diff
@@ -0,0 +1,4 @@
+stable:
+  sources-api:
+  manageiq-api-common:
```

Comment:
We are going to make a gem out of this, so I think we should remove it and release it.